testbed/tests/app/test_desktop.py

```diff
@@ -168,159 +170,183 @@ async def test_menu_minimize(app, app_probe):
     assert window1_probe.is_minimized
 
 
-async def test_full_screen(app, app_probe):
-    """Window can be made full screen"""
-    window1 = toga.Window("Test Window 1", position=(150, 150), size=(200, 200))
-    window2 = toga.Window("Test Window 2", position=(400, 150), size=(200, 200))
-
-    window1_widget = toga.Box(style=Pack(flex=1))
-    window2_widget = toga.Box(style=Pack(flex=1))
-    window1_widget_probe = get_probe(window1_widget)
-    window2_widget_probe = get_probe(window2_widget)
-
-    window1.content = toga.Box(
-        children=[window1_widget], style=Pack(background_color=REBECCAPURPLE)
+async def test_presentation_mode(app, app_probe, main_window, main_window_probe):
+    """The app can enter presentation mode."""
+    window_information_list = list()
+    windows_list = list()
+    for i in range(len(app.screens)):
+        window = toga.Window(title=f"Test Window {i}", size=(200, 200))
+        r = random.randint(0, 255)
+        g = random.randint(0, 255)
+        b = random.randint(0, 255)
+        window_widget = toga.Box(
+            style=Pack(flex=1, background_color=f"#{r:02X}{g:02X}{b:02X}")
+        )
+        window.content = window_widget
+        window.show()
+
+        window_information = dict()
+        window_information["window"] = window
+        window_information["window_probe"] = window_probe(app, window)
+        window_information["initial_content_size"] = window_information[
+            "window_probe"
+        ].content_size
+        window_information["widget_probe"] = get_probe(window_widget)
+        window_information["initial_widget_size"] = (
+            window_information["widget_probe"].width,
+            window_information["widget_probe"].height,
+        )
+        window_information_list.append(window_information)
```

Comment:
One property that isn't being validated here is the *screen* the window is on - this is something that is being explicitly set, but not actually confirmed anywhere. 

It's something that will be difficult to verify in CI, as CI only has one screen - but it would be worth checking regardless.

Reply:
I have added checks for screen in the test.